.travis.yml

```diff
@@ -112,11 +128,14 @@ script:
         ;
       fi
     - if [ "${TARGET}" != "${TARGET/e2e_third/}" ]; then
-        cd server &&
+        cd $TRAVIS_BUILD_DIR/server &&
+        python3 manage.py app:initialize_data;
         python3 manage.py users:create -u admin -p admin -e 'admin@example.com' --admin ;
+        python3 manage.py register_local_themes ;
+        python3 manage.py schema:migrate ;
         cd ../client &&
         ./node_modules/.bin/webdriver-manager update &&
-        ./node_modules/.bin/protractor-flake --node-bin node --max-attempts=5 -- protractor-conf.js --suite=c
+        ./node_modules/.bin/protractor-flake --node-bin node --max-attempts=1 -- protractor-conf.js --suite=c
```

Comment:
because session timeouts some test can fail and that is why `max-attempts=5` so that we don't manual restart test so often.